doc/make.jl

```diff
@@ -368,26 +368,60 @@ end
 
 const devurl = "v$(VERSION.major).$(VERSION.minor)-dev"
 
+function preRelease_versions(available_folders)
```

Comment:
Just a minor style nitpick, but we don't usually `camelCase` function names in Julia:

```suggestion
function prerelease_versions(available_folders)
```